chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2383,7 +2383,16 @@ public synchronized String sceneStatus() {
           buf.append("\n");
         }
         Vector3 pos = camera.getPosition();
-        buf.append(String.format("pos: (%.1f, %.1f, %.1f)", pos.x, pos.y, pos.z));
+        buf.append(String.format("pos: (%.1f, %.1f, %.1f)\n", pos.x, pos.y, pos.z));
+
+        buf.append("facing: ");
+        double yaw = camera.getYaw();
+        yaw = (yaw + Math.PI*2) % (Math.PI*2);
```

Comment:
What does this line do? Isn't it the same as `double yaw = camera.getYaw() % (Math.PI * 2)` ?

Reply:
`camera.getYaw()` ranges from `-2PI` to `2PI`. This was just the simplest way I could think of converting it to `0` to `2PI`. A simple modulo would not convert the negative angles to positive.

Reply:
Unrelated to this PR but that range doesn't make any sense :smile: 0…2π would be enough